.circleci/config.yml

```diff
@@ -53,6 +53,23 @@ executors:
     - image: *ci_support_image
 
```

Comment:
Add the following to the "references" section above (github won't let comment on that part of file):
```yaml
tcell_gcp_path: &tcell_gcp_path
            gs://ddp-tcell/tcell-1.10.0.tar.gz